include/cereal/archives/json.hpp

```diff
@@ -68,6 +68,11 @@ namespace cereal
 #include <vector>
 #include <string>
 
+#if defined(__clang__) or (defined(__GNUC__) and __GNUC__ >= 7)
```

Comment:
I think the `or` and `and` need to be changed to boolean operators (`||` and `&&`) to make MSVC happy